server/middleware/setUpHealthChecks.ts

```diff
@@ -21,7 +26,15 @@ export default function setUpHealthChecks(applicationInfo: ApplicationInfo): Rou
     }),
   )
 
-  router.get('/info', (req, res) => {
+  router.get('/info', async (req, res) => {
+    let activeAgencies: string[]
+    try {
+      activeAgencies = await supportedPrisonsService.getActiveAgencies()
+    } catch (error) {
+      logger.error(error, 'Error retrieving activeAgencies')
+      activeAgencies = []
+    }
+
```

Comment:
Key part of the change: fetch supported prison IDs from API. Catch errors and handle with empty array. (Did try using `asyncMiddleware` handler but if there's an API error you get HTML error page sent back....